pkg/virt-api/webhooks/validating-webhook/admitters/vmirs-admitter_test.go

```diff
@@ -113,18 +145,21 @@ var _ = Describe("Validating VMIRS Admitter", func() {
 				Selector: &metav1.LabelSelector{
 					MatchLabels: map[string]string{"match": "me"},
 				},
-				Template: newVirtualMachineBuilder().
-					WithDisk(v1.Disk{
-						Name: "testdisk",
-					}).
-					WithVolume(v1.Volume{
-						Name: "testdisk",
-						VolumeSource: v1.VolumeSource{
-							ContainerDisk: testutils.NewFakeContainerDiskSource(),
-						},
-					}).
-					WithLabel("match", "me").
-					BuildTemplate(),
+				Template: buildVMITemplate(
+					libvmi.New(
+						libvmi.WithName("testvmi"),
+						withDisk(v1.Disk{
+							Name: "testdisk",
+						}),
+						withVolume(v1.Volume{
+							Name: "testdisk",
+							VolumeSource: v1.VolumeSource{
+								ContainerDisk: testutils.NewFakeContainerDiskSource(),
+							},
+						}),
```

Comment:
Use `libvmi.WithContainerDisk` instead. That should allow you to drop `withVolume`.